### ISSUE TITLE
Kanban column titles inverted when collapsed

### ISSUE THREAD

**Issue by hhubbell** [odoo/odoo#10744]
## Issue
A recent Chrome update (v.48) introduced support for the CSS declaration `writing-mode: tb-rl` which allows text to be written at a 90 degree angle; see [Documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/writing-mode) for additional information.  This causes any collapsed kanban columns to become totally inverted due to rotating by 90 degrees again - see CSS excerpt below.  It is likely this has been broken starting in Firefox 45, which was released December 15th.
```css
.openerp .oe_kanban_view .oe_kanban_group_title_vertical {
    writing-mode: tb-rl;
    -webkit-transform: rotate(90deg);
    -moz-transform: rotate(90deg);
    -o-transform: rotate(90deg);
    -ms-transform: rotate(90deg);
    transform: rotate(90deg);
    width: 30px;
    font-size: 24px;
    white-space: nowrap;
    display: none;
    position: relative;
    opacity: 0.75;
    top: 26px;
}
```

Where these lines are the culprit:
```css
{
    writing-mode: tb-rl;
    ...
    transform: rotate(90deg);
}
```

## Solution
The simple solution would be to pick one or the other to rotate the text, but not both.

## Additional
Impacted versions:

 - 8.0

Current behavior:

 - Fields titles are inverted

Expected behavior:

 - Field titles are not inverted

**Comment by mmalorni** [comment]
Firefox 44 has the same issue but was fine in version 42.
IE 11.0.9600.18097 has the issue as well.
Safari 9.0.3 doesn't have the issue.

**Comment by xmo-odoo** [comment]
Are you sure 8.0 is impacted? There is a PR #10687 to change this in 7.0, but for 8.0 it should have been fixed by #7955, is your branch up to date? Grepping the current 8.0 code, `writing-mode` is only present in a pair of report templates.

**Comment by hhubbell** [comment]
My branch is out of date.  Applying 97dbe0db5f78032bb1eb83524a84563dc02b0c92 fixes the issue.

Thanks
